app/api/recall/bot/route.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from "zod";
+import { withApiWrapper } from "@/utils/api-wrapper";
+import { BadRequestError } from "@/server/errors";
+
+const createBotSchema = z.object({
+  meeting_url: z.string().url(),
+  bot_name: z.string().optional(),
+});
+
+async function createBotHandler(body: unknown) {
+  const { meeting_url, bot_name } = createBotSchema.parse(body);
+
+  const recallApiKey = process.env.RECALL_API_KEY;
+  if (!recallApiKey) {
+    throw new BadRequestError(
+      "Recall API key not configured. Please set RECALL_API_KEY environment variable."
+    );
+  }
```

Comment:
Query the applications table for the Zoom application, get the credential from there